conf/modules/controlfreec.config

```diff
@@ -50,7 +50,7 @@ process {
         ]
     }
 
-    withName: 'MAKEGRAPH' {
+    withName: 'MAKEGRAPH2' {
```

Comment:
do we still need the params.cf_ploidy? I thought makegrpah2 is able to infer ploidy?